kornia/geometry/quaternion.py

```diff
@@ -244,6 +244,27 @@ def from_matrix(cls, matrix: Tensor) -> 'Quaternion':
         """
         return cls(rotation_matrix_to_quaternion(matrix, order=QuaternionCoeffOrder.WXYZ))
 
+    @classmethod
+    def from_axis_angle(cls, axis: Tensor, angle: Tensor) -> 'Quaternion':
```

Comment:
Why not just call `angle_axis_to_quaternion` ? Also, why 2 inputs instead of angle * axis?
 https://github.com/kornia/kornia/blob/master/kornia/geometry/conversions.py#L786
 
 Also on the first

Reply:
Oops, didn't realize the conversion was already there! Yes, angle*axis seems to be a better representation!